deploy/push_to_rdss_willow.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+cd /home/willow/willow
+git push -u rdss-willow master
```

Comment:
I'm assuming you set the remote with sth like `git remote set-url` manually on the server